pkg/trace/api/api.go

```diff
@@ -83,6 +84,10 @@ type HTTPReceiver struct {
 func NewHTTPReceiver(
 	conf *config.AgentConfig, dynConf *sampler.DynamicConfig, out chan pb.Trace, services chan pb.ServicesMetadata,
 ) *HTTPReceiver {
+	presamplerResponse := http.StatusOK
+	if config.HasFeature("429") {
```

Comment:
Let's be sure we make this the default behavior in a newer API for the next release.